contracts/id.tonomy/src/id.tonomy.cpp

```diff
@@ -49,7 +49,7 @@ namespace idtonomy
       return name(name_string);
    }
 
-   name random_account_name(const checksum256 &hash1, const checksum256 &hash2)
+   name random_account_name(const checksum256 &hash1, const checksum256 &hash2, const char &account_type)
```

Comment:
better if this accepts `enum_account_type` as argument to control the letter, not the letter itself. Otherwise the caller of the function can choose any letter.

Reply:
updated type to enum